.travis.yml

```diff
@@ -1,12 +1,22 @@
 sudo: false
+os:
+  - linux
+  - osx
 language: node_js
 node_js:
+  - node
   - '6'
   - '5'
-  - '4'
```

Comment:
0.10 was the mainstream version for a long time, and many enterprise users are still stuck on it. We want to keep supporting it for a while.